LICENSE

```diff
@@ -0,0 +1,8 @@
+YEAR: 2019
+COPYRIGHT HOLDER: OHRI
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
There are no conditions, after "subject to the following conditions:  "

Reply:
Are there suppose to be any? I obtained file from here: https://opensource.org/licenses/MIT

Reply:
OK. I understand now. The next sentence is the only condition.